src/main/java/seedu/address/logic/commands/CommandWord.java

```diff
@@ -21,6 +22,9 @@
     LIST("list"),
     VIEW("view"),
     SORT("sort"),
+    REMIND("remind"),
+    REMINDER("listr"),
+    SHOW_REMINDER("showr"),
```

Comment:
Merged from another branch? They should'nt ber here for this PR. This means that the other PR has to be approved first before this. Else might as well have this PR only.